components/parallel-libs/trilinos/SPECS/trilinos.spec

```diff
@@ -15,32 +15,38 @@
 
 # Base package name
 %define pname trilinos
-%define ver_exp 12-18-1
 
 Name:           %{pname}-%{compiler_family}-%{mpi_family}%{PROJ_DELIM}
-Version:        12.18.1
+Version:        13.0.0
 Release:        1%{?dist}
 Summary:        A collection of libraries of numerical algorithms
-License:        LGPL-2.0
+# Trilinos is licensed on a per-package basis. Refer to https://trilinos.github.io/license.html
+License:        LGPL-2.1 and BSD-3-Clause
 Group:          %{PROJ_NAME}/parallel-libs
 Url:            https://trilinos.org/
+%global         ver_exp %{lua: print((string.gsub(rpm.expand("%{version}"),"%.","-")))}
```

Comment:
It would be helpful to build under OBS before submitting. These lua_expansions are incompatible with the automatic tarball download facility of OBS. Please stop adding them in as I have to take them out each time.